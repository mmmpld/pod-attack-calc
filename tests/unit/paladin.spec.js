import { describe, it } from 'vitest'
import breakpoints from './breakpoints.js'
const characterId = 5

describe('Paladin default values', () => {
  it('standard attack', async () => {
    await breakpoints('13', '1.92', { charactersSelected: characterId })
  })
})
