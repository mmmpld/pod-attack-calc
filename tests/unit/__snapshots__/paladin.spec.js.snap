// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paladin default values standard attack 1`] = `
<tbody>
  <tr class="highlight-current">
    <td>0</td>
    <td>13</td>
    <td>1.92</td>
  </tr>
  <tr class="">
    <td>9</td>
    <td>12</td>
    <td>2.08</td>
  </tr>
  <tr class="">
    <td>20</td>
    <td>11</td>
    <td>2.27</td>
  </tr>
  <tr class="">
    <td>37</td>
    <td>10</td>
    <td>2.5</td>
  </tr>
  <tr class="">
    <td>63</td>
    <td>9</td>
    <td>2.77</td>
  </tr>
  <tr class="">
    <td>105</td>
    <td>8</td>
    <td>3.12</td>
  </tr>
  <tr class="">
    <td>200</td>
    <td>7</td>
    <td>3.57</td>
  </tr>
</tbody>
`;
